bootstrap/sql/com.mysql.cj.jdbc.Driver/v006__create_db_connection_info.sql

```diff
@@ -9,4 +9,7 @@ CREATE TABLE IF NOT EXISTS web_analytic_event (
     deleted BOOLEAN GENERATED ALWAYS AS (json -> '$.deleted'),
     UNIQUE(name),
     INDEX name_index (name)
-);
\ No newline at end of file
+);
+
+UPDATE bot_entity
+SET json = JSON_INSERT(JSON_REMOVE(json, '$.botType'), '$.provider', 'system');
```

Comment:
This is not covering the case where the user has already created some bots. It assumes all the bots in the DB are the system's bots.